drivers/wifi/nrf700x/Kconfig

```diff
@@ -151,6 +151,10 @@ config NRF700X_DATA_TX
 	bool "Enable TX data path in the driver"
 	default y if NRF700X_SYSTEM_MODE
 
+config NRF700X_RAW_DATA_TX
+	bool "Enable RAW TX data path in the driver"
+	default y if NRF700X_STA_MODE
```

Comment:
Why do we want to enable this by default?

Reply:
disabled default setting